### ISSUE TITLE
Please provide example for the API

### ISSUE THREAD

**Issue by leninchris** [najeal/react-native-autocomplete-search#27]
<InputAutoSuggest
  style={{ flex: 1 }}
  apiEndpointSuggestData={text => YOUR-METHOD-CALLING-API(text)}
  itemFormat={{id: 'details.id', name: 'details.name', tags:['details.continent', 'details.country']}}
  onDataSelectedChange={data => console.log(data)}
/>

Please provide an example.. because the data not getting loaded when triggered from API

**Comment by leninchris** [comment]
Any update on the issue ?

**Comment by ghost** [comment]
apiEndpointSuggestData not woking

**Comment by sevaa** [comment]
`apiEndpointSuggestData` should be a function that returns a promise that resolves to an object containing an async emethod called "json()". That method, in turn, should return a promise resolving to an object that contains a property with the name that corresponds to the value of the `keyPathRequestResult` property of your `InputAutoSuggest`. The content of said property should be an array of objects with `id` and `name` in them.

The default value of `keyPathRequestResult` is a bogus one, obviously one from a sample that the maintainers forgot to replace with a sensible default for a real app.

---

The flow within the search control goes:

```
  const response = await apiEndpointSuggestData(text);
  const responseData = await response.json();
  return suggestFormat(_.get(responseData, keyPathRequestResult), itemFormat, text);
```
Where _ is lodash.

So, to recap:
 - apiEndpointSuggestData should be async and return something with a `json()` method in it
 - `json()` should be async and return something with a property that contains an array of items
 - the name of that property should be passed to the control as `keyPathRequestResult`
